### PR TITLE
add a flag to control whether update the tiered locality configmap content or not

### PR DESCRIPTION
**Pull request by xliuqq**
### Ⅰ. Describe what this PR does
Add a flag to control whether update the tiered locality configmap content or not

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/fluid/fluid/values.yaml [hunk 1]
```diff
@@ -149,6 +149,8 @@ webhook:
   reinvocationPolicy: IfNeeded
   tolerations:
     - operator: Exists
+  # if configmap `tiered-locality-config` exists and not want to replace the content, set this to false.
+  updateTieredLocality: true
```
Comment (cheyang) [inline charts/fluid/fluid/values.yaml:153]: How about changing `updateTieredLocality` to `forceReplaceTieredLocality`? And by default, the value is false. Thanks.

File: charts/fluid/fluid/values.yaml [hunk 2]
```diff
@@ -149,6 +149,8 @@ webhook:
   reinvocationPolicy: IfNeeded
   tolerations:
     - operator: Exists
+  # if configmap `tiered-locality-config` exists and not want to replace the content, set this to false.
+  forceReplaceTieredLocality: true
```
Comment (cheyang) [inline charts/fluid/fluid/values.yaml:153]: I suggest changing the default value to false. Thanks.

Comment (xliuqq) [reply charts/fluid/fluid/values.yaml]: sorry to forget it, fixed.

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**